group-generators/helpers/data-providers/alchemy/index.ts

```diff
@@ -191,37 +191,37 @@ export class AlchemyProvider {
   }
 
   /**
-   *Fetches owners of NFTs with a matching tokenid
+   *Fetches owners of NFTs with matching tokenids. This is used internally and is very verbose.
    * @param {Object} params - The parameters for fetching owners by token IDs.
    * @param {string} contractAddress - The address of the contract to query.
    * @param {string} chain - The blockchain network to query.
-   * @param {string[]} tokenIds - An array of ERC-1155 token IDs to fetch owners for.
+   * @param {string[]} tokenIds - An array token IDs to fetch owners for.
    * @returns {Promise<FetchedData>} - A promise that resolves to an object mapping owner addresses to the number of token balances they hold.
    */
-  private async getOwnersByTokenIds({
+  private async getMetadataFromNftTokenIds({
     contractAddress,
     chain,
     tokenIds,
-  }: GetOwnersByTokenIdsParams): Promise<FetchedData> {
+  }: GetMetadataFromNftTokenIdsParams): Promise<FetchedData> {
     if (contractAddress && chain) {
       this.baseUrl = this.urlQueryHandler(chain);
       this.contractAddress = contractAddress;
     }
 
     const owners: OwnerInfo[] = [];
-    for await (const owner of this._getOwnersByTokenIds()) {
+    for await (const owner of this._getMetadataFromNftTokenIds()) {
       owners.push(owner);
     }
 
     return this.filterOwnersByTokenId(owners, tokenIds);
   }
 
-  public async getOwnersByTokenIdsCount({
+  public async getMetadataFromNftTokenIdsCount({
```

Comment:
if we put `getMetadataFromNftTokenIds` in private, no need to have a count function (because we don't want to integrate the function in the Factory)

Reply:
lol, thanks for pointing that out. I'll update it